frontend/src/lib/components/LemonBubble/LemonBubble.scss

```diff
@@ -0,0 +1,93 @@
+@import '~/vars';
+
+$lemon-counter-small: 1.2rem;
+$lemon-counter-font-small: 0.6rem;
+$sf-small: 0.8;
+$sf-large: 1.2;
```

Comment:
What is San Francisco doing here? 😅

Reply:
that shorthand for `scale-factor` might have been too short ;)